Description: Add Boost -mt decoration, to ease backporting to Lenny
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2011-01-17
--- a/src/makefile.unix
+++ b/src/makefile.unix
@@ -17,10 +17,10 @@
 
 # for boost 1.37, add -mt to the boost libraries
 LIBS= \
-   -l boost_system \
-   -l boost_filesystem \
-   -l boost_program_options \
-   -l boost_thread \
+   -l boost_system-mt \
+   -l boost_filesystem-mt \
+   -l boost_program_options-mt \
+   -l boost_thread-mt \
    -l db_cxx \
    -l ssl \
    -l crypto \
